common/lib/xmodule/xmodule/js/src/html/edit.js

```diff
@@ -100,6 +100,7 @@
           theme: "modern",
           skin: 'studio-tmce4',
           schema: "html5",
+          entity_encoding: "raw",
```

Comment:
will the previously incorrectly saved links require a re-save/re-edit?

Reply:
@awaisdar001

Reply:
Let me verify.

Reply:
We would need a re-save because the encoded data is saved in the database and that data is used for display.

Reply:
Ok. We will inform this on CR ticket once the PR lands production.

Reply:
Sure sounds good.